src/components/WebcamCapture/WebcamCapture.js

```diff
@@ -77,13 +72,14 @@ class WebcamCapture extends Component {
 		if (this.state.cameraConnected) {
 			return (
 				<div>
+					<header>Hold up your snack to the camera</header>
 					<Webcam
 						audio={false}
 						height={height}
 						ref={this.webcam}
 						screenshotFormat="image/jpeg"
 						width={width}
-						videoConstraints={videoConstraints}
+						// videoConstraints={videoConstraints}
```

Comment:
Is this needed? If not can we remove the comment? We can use version control instead of comments for reverting this